x-pack/plugins/security_solution/server/lib/siem_migrations/rules/data/rule_migrations_data_rules_client.ts

```diff
@@ -42,6 +42,7 @@ export interface RuleMigrationFilters {
   status?: SiemMigrationStatus | SiemMigrationStatus[];
   ids?: string[];
   installable?: boolean;
+  prebuiltRulesOnly?: boolean;
```

Comment:
Why _Only_? we are not using _Only_ in the other filters: _statusOnly_, _idsOnly_, _installableOnly_...
```suggestion
  havePrebuiltRuleId?: boolean;
```

Reply:
done